.pipelines/templates/template-push-images-to-acr-tagged.yml

```diff
@@ -10,5 +10,17 @@ steps:
     az acr login --name "$RP_IMAGE_ACR"
     # azure checkouts commit, so removing master reference when publishing image
     export BRANCH=$(Build.SourceBranchName)
-    make publish-image-aro-tag
+    make publish-image-aro
```

Comment:
The change is in the makefile, which now identifies if the TAG was used, otherwise uses COMMIT